src/doc/unstable-book/src/language-features/intrinsics.md

```diff
@@ -24,6 +24,13 @@ fn foo() -> u32 {
 }
 ```
 
+## Intrinsics lowered to MIR instructions
+
+Various intrinsics have native MIR operations that they correspond to. Instead of requiring
```

Comment:
The other two sections (Intrinsics {with, without} fallback logic) show how each respective kind of intrinsic is declared. Can you give a hint here as to how an Intrinsic lowered-to-MIR-instructions is declared?